docs/source/Usage.rst

```diff
@@ -11,14 +11,24 @@ The following packages need to be installed for generating fabric HDLs
 
 :Python:
 
-Version >= 3.9
+Version >= 3.12
```

Comment:
I think we should specifiy the version that we use for the GitHub actions (GHA) since this version is guaranteed to be working. We could say something like "older versions might also work but are not officially supported" or we could also explcitely say ">= 3.10 might also work but is not officially supported". The alternative would be keep using 3.10 in the GHA and also change all docs and checks to 3.10 since we cirrently don't use features of >=3.11 anyway.

Reply:
Feature like fstring improvement and parametric typing might be useful later. 

How about we do official 3.12, so CI also on 3.12, Then saying 3.10, 3.11 is unofficial supported, but also should work. In the toml we do requirements as >=3.12. We should also remove the version check in the shell since the install will handle version checks.

Reply:
Sounds good to me, I will adjust it tomorrow!